lib/crypto/dsa_test.go

```diff
@@ -0,0 +1,28 @@
+package crypto
+
+import (
+  "crypto/dsa"
+  "crypto/rand"
+  "testing"
+)
+
+
+func TestDSA(t *testing.T) {
```

Comment:
Do you have an option regarding go's built in testing tools vs. frameworks like ginkgo?  I generally find BDD convenient but I'm not married to it.